src/App.tsx

```diff
@@ -1,61 +1,137 @@
 import './App.scss';
+import './field.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [correctTitle, addTitle] = useState(false);
+  const [userId, addUserId] = useState(0);
+  const [chosenUser, chooseUser] = useState(false);
```

Comment:
```suggestion
  const [correctTitle, setCorrectTitle] = useState(false);
  const [userId, setUserId] = useState(0);
  const [chosenUser, setChosenUser] = useState(false);
```

Reply:
const [value, setValue] = useState()

That is the only naming pattern for react state values

Reply:
OK, thanks. Fixed